Pages/LandingPage.py

```diff
@@ -0,0 +1,92 @@
+import csv
+import os
+
+import requests
+from bs4 import BeautifulSoup
+from selenium.webdriver.common.by import By
+
+from Constans import element_tuples, uri, arrays
+
+from Utilities.SetupClass import SetupClass
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class LandingPage:
+    scraped_array = []
+
+    def __init__(self, driver, wait, log):
+
+        self.driver = driver
+        self.wait = wait
+        self.log = log
+
+    def exit_script(self):
+        self.driver.quit()
+        os._exit(0)
```

Comment:
kodel po kiekvieno metodo iseini is skripto?

Reply:
ne po kiekvieno. Iseinu iskirtinai tada kai scriptas sufailina pvz nenuscrapina duomenu, nepasirenka Lt ir t.t. (bent jau intencija buvo tokia).